### PR TITLE
fix(curriculum): use get computed styles in step 46 of the access quiz

### LINKED ISSUE #52410: Accessibility Quiz step 46 should accept multiple valid CSS selectors
### Describe the Issue

Multiple CSS selectors are valid for this challenge, but only `li > a` passes the tests.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-46

### Your code

```css
nav > ul > li > a {
  color: #fff;
}
```

### Expected behavior

My code passes the tests.

### Screenshots

_No response_

### System

- Device: Laptop
- OS: Windows 11
- Browser: Edge
- Version: 119

### Additional context

Since the provided answer for the previous challenge is `nav > ul > li`, the logical answer for this challenge should be `nav > ul > li > a`. But it's not accepted.

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #52410

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md [hunk 1]
```diff
@@ -16,13 +16,16 @@ Change the font color of all the anchor elements within the list elements to som
...
-assert.exists(new __helpers.CSSHelp(document).getStyle('li > a'));
+const anchors = document.querySelector('li > a'); 
+assert.exists(getComputedStyle(anchors));
 ```
 
 You should give the `a` element a `color` property.
 
 ```js
-assert.notEmpty(new __helpers.CSSHelp(document).getStyle('li > a')?.color);
+const anchors = document.querySelector('li > a');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md:26]: this gets only the first `li > a`, not all of them. Maybe you can use `querySelectorAll`, and check that the style exist for each `li > a`? Or is it unnecessary?

Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md]: I probably should check all of them.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md [hunk 2]
```diff
@@ -16,13 +16,16 @@ Change the font color of all the anchor elements within the list elements to som
 You should use the `li > a` selector.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('li > a'));
+const anchors = document.querySelectorAll('li > a'); 
+assert.exists(getComputedStyle(anchors));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md:20]: now my question is, does `getComputedStyle` works on a NodeList?
shouldn't you do like `anchors.forEach(a => assert.exists(getComputedStyle(a))`
note, this is a genuine question, I am just wondering

File: .husky/pre-commit
Comment (ilenia-magoni) [inline .husky/pre-commit:1]: does this file belong to the PR?

Comment (a2937) [reply .husky/pre-commit:1]: The Husky pre-commit file kept losing the ability to execute on my WSL instance and every time I made it executable again; it keeps trying to re-add that permission into my pull requests. 

It has absolutely nothing to do with this pull request.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md (hunk 2 above)
Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md:20]: I was a little distracted but it only takes a single element. At least I learned something new. 

 https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle

Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md]: Okay this seems to have been fixed.

File: .husky/pre-commit
Comment (a2937) [reply .husky/pre-commit:1]: I'm having trouble finding an article on how to properly remove a file from the changes in a Git PR. Hope this is close enough.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md [hunk 3]
```diff
@@ -16,13 +16,15 @@ Change the font color of all the anchor elements within the list elements to som
 You should use the `li > a` selector.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md:16]: Now I wonder, considering that this selector is not compelled anymore, should the hint be changed?

Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md:16]: It still is an important to solve the problem though.  Campers can add stuff on top of it, but it is still fundamentally an `li > a` selector.

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by moT01 — APPROVED** [review]
LGTM @a2937 🎉

**Pull request merged**